DirectProgramming/C++SYCL_FPGA/Tutorials/Features/IPAuthoring_interfaces_overview/naive/README.md

```diff
@@ -0,0 +1,184 @@
+# Naive Design
+This example demonstrate a naive implementation of a simple vector addition kernel. 
+
+This design will show interfaces that is invoked by default by the Intel icpx compiler.
```

Comment:
This simple vector add implementation is similar to the one presented in the [fpga_compile](LINK PLEASE) code sample. It demonstrates a kernel that uses a register-mapped invocation interface, and a single shared memory-mapped host data interface. These are the default interfaces that are selected by the Intel® oneAPI DDPC++/C++ Compiler, and therefore this is the only design that can support the full-system compilation flow.